README.md

```diff
@@ -100,7 +100,7 @@ key is in this repo under [deb.asc](/deb.asc).
 
 To install eza from this repo use:
 ```bash
-wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | sudo tee /etc/apt/trusted.gpg.d/gierens.asc
+wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/gierens.asc
```

Comment:
Hm, are we really sure that with `| gpg --dearmor |` it works everywhere? Because the person that reported the need to dearmor used this in a Docker container which is quite an edge case. For everyone else it seems to have worked with the ASCII key file, I wouldn't wanna break it for them because of one edge case.

Therefore my plan was actually to add a comment hinting towards using the dearmor pipe when necessary, but leaving the suggested command as is.

Reply:
I mean if you are sure that it works everywhere it's fine with me, but I never used dearmor before and just wanna make sure that the suggested command works for the average user.

Reply:
Ok let me test it and we'll make sure